editor/editor_help_search.cpp

```diff
@@ -651,23 +748,19 @@ TreeItem *EditorHelpSearch::Runner::_create_theme_property_item(TreeItem *p_pare
 }
 
 TreeItem *EditorHelpSearch::Runner::_create_member_item(TreeItem *p_parent, const String &p_class_name, const String &p_icon, const String &p_name, const String &p_text, const String &p_type, const String &p_metatype, const String &p_tooltip, bool is_deprecated, bool is_experimental) {
-	Ref<Texture2D> icon;
-	String text;
+	const String item_meta = "class_" + p_metatype + ":" + p_class_name + ":" + p_name;
+
+	TreeItem *item = _find_or_create_item(p_parent, item_meta);
+	item->set_icon(0, ui_service->get_editor_theme_icon(p_icon));
 	if (search_flags & SEARCH_SHOW_HIERARCHY) {
-		icon = ui_service->get_editor_theme_icon(p_icon);
-		text = p_text;
+		item->set_text(0, p_text);
 	} else {
-		icon = ui_service->get_editor_theme_icon(p_icon);
-		text = p_class_name + "." + p_text;
+		item->set_text(0, p_class_name + "." + p_text);
 	}
```

Comment:
Same here, except for this bit which needs to be updated accordingly, of course.